upup/models/cloudup/resources/addons/aws-ebs-csi-driver.addons.k8s.io/k8s-1.17.yaml.template

```diff
@@ -726,6 +726,12 @@ spec:
         - effect: NoExecute
           operator: Exists
           tolerationSeconds: 300
+        {{- if KarpenterEnabled }}
```

Comment:
We should discuss in office hours; my preference is to keep the "business logic" out of the template and in go code, so something like "if RunOnControlPlane".  But I fully ack that this is just my preference and the way we/I set up template functions as one big shared pool does not make this as nice in practice

Reply:
This is from many years, but I agree it should be changed into something more generic.